docs/src/docs/asciidoc/getting-started.adoc

```diff
@@ -146,12 +146,16 @@ the configuration are described in the following listings:
 .Gradle
 ----
 	plugins { <1>
-		id "org.asciidoctor.convert" version "1.5.9.2"
+		id "org.asciidoctor.jvm.convert" version "3.3.2"
+	}
+
+	configurations {
+		asciidoctorExt
 	}
 
 	dependencies {
-		asciidoctor 'org.springframework.restdocs:spring-restdocs-asciidoctor:{project-version}' <2>
-		testImplementation 'org.springframework.restdocs:spring-restdocs-mockmvc:{project-version}' <3>
+		testImplementation 'org.springframework.restdocs:spring-restdocs-mockmvc:{project-version}' <2>
+		asciidoctorExt 'org.springframework.restdocs:spring-restdocs-asciidoctor:{project-version}' <3>
```

Comment:
I'd prefer to keep the dependency declarations in their current order please. This will mean that the changes to the order of the callouts can also be reverted.

Reply:
Done!